src/swsssdk/scripts/sonic-db-cli

```diff
@@ -36,3 +59,37 @@ else:
             print "\n".join(resp)
         else:
             print resp
+        sys.exit(0)
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                                     formatter_class=argparse.RawTextHelpFormatter,
+                                     epilog=
+"""
+Example 1: sonic-db-cli -n asic0 CONFIG_DB keys *
+Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
+Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
+Example 4: sonic-db-cli -n asic3 APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+Example 5: sonic-db-cli PING
+""")
+    parser.add_argument('-n', '--namespace', type=str, help="Namespace string to use asic0/asic1.../asicn", default=None)
+    parser.add_argument('db_or_op', type=str, help='Database name Or Unary operation(only PING supported)')
+    parser.add_argument('cmd', nargs='*', type=str, help='Command to execute in database')
+
+    args = parser.parse_args()
+
+    if args.db_or_op:
+        if args.cmd:
+            execute_cmd(args.db_or_op, args.cmd, args.namespace)
+        elif args.db_or_op == 'PING':
+            ping_all_instances()
```

Comment:
add namespace parameter to get correct instancelist ?

Reply:
shall update, I skipped earlier as these were new functions.